.travis.yml

```diff
@@ -37,3 +37,13 @@ notifications:
     on_failure: change
     template:
       - "%{repository_slug}#%{build_number} (%{branch} - %{commit} : %{author}): %{message} %{build_url}"
+
+deploy:
+  provider: pypi
+  user: "foobar"
```

Comment:
**TODO**: we'll need to encrypt pypi auth and update these vars. 
see: https://docs.travis-ci.com/user/deployment/pypi/

Reply:
What would be a good way to do this safely?

Reply:
Aha!  Just started reading what you linked to ^_^